src/index.ts

```diff
@@ -217,18 +217,17 @@ export class DID {
     if (typeof jws !== 'string') jws = fromDagJWS(jws)
     const kid = base64urlToJSON(jws.split('.')[0]).kid as string
     if (!kid) throw new Error('No "kid" found in jws')
-    const { publicKey } = await this.resolve(kid)
+    const didResolutionResult = await this.resolve(kid)
+    const publicKeys = didResolutionResult.didDocument!.verificationMethod || []
```

Comment:
As noted by the typecheck warning below, it seems unsafe as `didDocument` could be `null` according to this type: https://github.com/decentralized-identity/did-resolver/blob/master/src/resolver.ts#L17

Reply:
It will really only be `null` if there is an error, which is caught in the `this.resolve` method. Maybe I can improve it there?

Reply:
Updated to use `?` instead.